framework/src/governance/lib/data-lake-catalog.ts

```diff
@@ -10,13 +10,14 @@ import { AnalyticsBucket } from '../../storage';
 import { Context, TrackedConstruct, TrackedConstructProps } from '../../utils';
 
 /**
-* Creates AWS Glue Catalog Database for each storage layer. Composed of 3 {@link DataCatalogDatabase} for Bronze, Silver, and Gold data.
+* Creates a Data Lake Catalog on top of a `DataLakeStorage`.
+* The Data Lake Catalog is composed of 3 `DataCatalogDatabase`, one for each storage layer.
 * @see https://awslabs.github.io/data-solutions-framework-on-aws/docs/constructs/library/Governance/data-lake-catalog
  *
 * @example
```

Comment:
I thought we agreed to remove all examples from construct hub and move them to the website doc